academic-services/knowledge-exploration-service/get-started-create-api-instances.md

```diff
@@ -5,24 +5,79 @@ ms.topic: tutorial
+
+- Open the 'dated' folder, then verify there is a sub folder named 'tools'.
+
+![Verify tools folder](media/get-started-tools-folder.png)
+
+## Download the deployment script from your Azure Storage Account
+
+Each MAKES deployment includes the scripts required to provision an instance of MAKES in Azure.  These tools are located in the 'tools' folder that you verified above.  To download the scripts, select the 'kesm.zip' file from Storage Explorer and click 'Download' from the top ribbon:
+
+![Download kesm.zip](media/get-started-download-kesm.png)
+
+Save this file to a local folder.
+
+## Unzip the script package
+
+Once the file has been downloaded you will need to extract the contents.  Right click on the kesm.zip file.  If you are using the Windows operating system select "Extract All" from the context menu.  If you are using linux, select "Extract Here" or "Unzip Here".  If you are using a Mac, double-click the zip file and it will decompress into the same folder.
+
+If this has completed successfully you will have extracted a single file named kesm.exe.
+
+## Run the script
+
+At this point to are ready to deploy an instance of MAKES to your Azure account.  Open a command prompt (Windows) or terminal window (Mac) and navigate to the folder that you extracted the kesm.exe file to.
 
 Execute the following command to deploy hosting resources
 
 ```cmd
-makes.exe CreateHostResource --MakesPackageBaseDir "https://[storage account name].blob.core.windows.net/makes/[release version]/"
+kesm.exe DeployHost --HostName [whatever_name_you_would_like] --MakesPackage "https://[your_storage_account_name].blob.core.windows.net/makes/[release version]/"
+```
+
+- Replace "[whatever_name_you_would_like]" with the host name for your service.  The hostname will be the host name of the server where your MAKES deployment will be hosted.  Ex: If you used 'testmakes01102020', your MAKES API will be hosted at http://testmakes10012020.westus.cloudapp.azure.net
+
+- Replace "[your_storage_account_name]" with the name of the storage account you downloaded the scripts from above.
+
+At this point the tool will take care of creating all of the required resources and deploying your instance of MAKES.  This process can take a long time.  During the initial run it can take up to 3 hours to complete.  During the first run the script generates some resources that you only need to create once.  Subsequent deployments will not take as long if you specify existing resources.  See the [Command line Reference](reference-makes-command-line-tool.md) for more details.  
+
+Depending on the region your storage account is in and the region you are deploying to, deployment may take longer as the default indexes are quite large and need to be copied.  By default, the tool deploys to the WestUS region of Azure.  You can change this by adding parameters to the script.  For a reference of all the available parameters type:
```

Comment:
I think this section should be tagged with as an "important" section. I'd suggest the following:

[!NOTE] To achieve the fastest instance start times ensure that all resources (storage account, virtual machine scale set, etc.) are located in the same region. The "--Region" parameter controls which region new resources are created in. Visit the [Command Line Reference](reference-makes-command-line-tool.md) section for full details on this and other parameters.

Reply:
I was wrong on the formatting for these, sorry!

It needs to be:
> [!NOTE|IMPORTANT|WARNING|INFO]
> Message

---
In reply to: [370773816](https://github.com/MicrosoftDocs/microsoft-academic-services/pull/90#discussion_r370773816) [](ancestors = 370773816)